crates/scope/src/builder.rs

```diff
@@ -2490,7 +2506,7 @@ impl ScopeDataMapBuilder {
         if self.scope_kind_stack.is_empty() {
             // FIXME
             // Do nothing for unsupported case.
-            // Emitter will return NotImplemented anyway.
+            self.set_not_implemented("Unsupported binding identifier");
```

Comment:
nit: update https://github.com/mozilla-spidermonkey/jsparagus/blob/master/.metrics/populate_not_implemented.sh#L23 to account for these new `not_implemented` cases.

Reply:
I'll change those lines to directly instantiate NotImplemented